### PR TITLE
docs: Add more details regarding Linux podman machine

### PR DESCRIPTION
**Pull request by cdrage**
docs: Add more details regarding Linux podman machine

### What does this PR do?

Adds some more details regarding rootful mode on Linux with podman
machine. Since you cannot create the machine through the podman desktop
GUI, you must create it via the command line.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

References #93

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,15 @@ Or set when initially creating a Podman Machine via Podman Desktop:
 
 ![rootful setup](/docs/img/rootful_setup.png)
 
+**Linux users:** 
+
+On Linux, you are unable to create a Podman Machine through the GUI of Podman Desktop, to create a rootful Podman Machine you can run the following commands:
+
+```sh
+podman machine init --rootful-true
```
Comment (benoitf) [inline README.md:112]: are you sure of this syntax ?

**Comment by cdrage** [comment]
@benoitf Updated, sorry about that. Should of been `--rootful` not `--rootful-true`

**Review by benoitf — APPROVED** [review]

**Review by vrothberg — APPROVED** [review]
LGTM, thank you!

**Review by slemeur — APPROVED** [review]

**Review by benoitf — COMMENTED** [review]
@cdrage merge ?

**Pull request merged**